src/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ target_sources(sm PRIVATE
     "snes/snes.c"
     "snes/snes_other.c"
     "snes/spc.c"
-)
+ "extra_features.h")
```

Comment:
Headers don't need to be included in `target_sources()`, it's added below by `target_include_directories()`

Reply:
Silly VS added that and I just didn't pay attention it is now removed.